src/components/SignIn.js

```diff
@@ -16,6 +17,10 @@ const SignInPage = ({ history }) =>
     </div>
   );
 
+SignInPage.propTypes = {
```

Comment:
React patterns introduce to keep only one component in a file. Split it on two files to keep it simple :)